web/modules/custom/dept_migrate/src/MigrateUuidLookupManager.php

```diff
@@ -139,6 +140,55 @@ public function lookupBySourceNodeId(array $nids) {
     return $map;
   }
 
+  /**
+   * @param array $uids
+   *   One or more source user ids.
+   *
+   * @return array
+   *   Associative array of user metdata, keyed by source user id
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  public function lookupBySourceUserId(array $uids) {
```

Comment:
I get the feeling these can be reworked to be more general and by passing in the entity type and ids needed. That said, they're working for now, so I'm minded to carry on with a more verbose but more literal approach on these. This will be a throwaway class for supporting the migrations so efficiency isn't a huge priority right now.